browser/components/preferences/in-content/main.js

```diff
@@ -143,7 +146,13 @@ var gMainPane = {
       alertsService.showAlertNotification("",  "Restart Waterfox", "You'll need to restart Waterfox to see your selected locale.");
     }
   },
-
```

Comment:
Special event function that will call `updateLocale` when the **ENTER (Return Key)** is pressed